linkinator.remote.config.json

```diff
@@ -2,18 +2,19 @@
   "__available_flags": "https://github.com/JustinBeckwith/linkinator/blob/main/src/config.ts#L3",
   "recurse": true,
   "skip": [
-    "/.+/(page-data|static)/(.*)",
+    "/.+/(page-data|static|icons)/(.*)",
     "/[^/]+/[^/]+.(js|json)$",
+    "/.+/favicon(.*)",
     "https://support.commercetools.com",
-    "/api$",
-    "/api/",
-    "/docs/",
-    "/sdk",
-    "/tutorials",
-    "/custom-applications",
-    "/merchant-center",
-    "/getting-started",
-    "/import-export"
+    "/.+/api$",
+    "/.+/api/",
+    "/.+/custom-applications",
+    "/.+/docs/",
+    "/.+/import-export",
+    "/.+/merchant-center",
+    "/.+/sdk",
+    "/.+/tutorials",
+    "/.+/getting-started"
```

Comment:
Just curious: What problem does this address?  Looks like a workaround to me, do we have `/../api` style link hrefs in the final static output?

Reply:
I found out that paths like `/docs/` interfere with the domain (for example `https://docs-kit`), resulting in all links to be ignored.

Adding the `.+` seems to fix that. So for consistency I added it to all paths.

Reply:
uh, ugly edge case.  Thanks.